### PR TITLE
feat: abstract captureOptions() to MixCommand superclass

### PR DESCRIPTION
**Pull request by oliver-daniel**
This PR documents my progress in using the `this.constructor` accessor in `MixCommand` to abstract away the repetitive implementation of `captureOptions()` in (basically) every command.

Those commands that require different or additional behaviour after parsing use a `super` call in order to guarantee that future changes to `captureOptions()` are recorded downstream.

Tests continue to run nominally on my machine\, inexplicably, however, test coverage percentage went *down* across all files.

### CODE REVIEW

**Comment by oliver-daniel** [comment]
NB: In order to make the `no-extraneous-imports` ESLint rule happy (and in the interest of good code), I had to register `@oclif/parser` as a dependency in `package.json`. However, as discussed in #11, future changes are likely to lead to migrating these dependencies into the newer `@oclif/core`.

File: src/commands/app-configs/create.ts [hunk 1]
```diff
@@ -83,9 +83,7 @@ should be unique.`
   }
 
   captureOptions() {
-    debug('captureOptions()')
-    const {flags} = this.parse(AppConfigsCreate)
-    this.options = flags
+    super.captureOptions()
     debug('flags: %O', flags)
```
Comment (oliver-daniel) [inline src/commands/app-configs/create.ts:87]: This is debugging information which is replicated, at least partially, in `MixCommand`. Unless it's needed for a more delicate operation, this method can be deleted.

Comment (grof) [reply src/commands/app-configs/create.ts]: I removed the extra debug logs. Thanks!

**Review by grof — APPROVED** [review]
Looks good.

**Pull request merged**